### DISCUSSION TITLE
7z not exe for Windows Release

### DISCUSSION THREAD

**Discussion by Andystew94** [mamedev/discussions#82] [General]
Hi, I am wondering if the latest release on GitHub "mame0261b_64bit.exe" is meant to have the .7z file extension, not .exe?

**Comment by cuavas** [comment]
It should be self-extracting on Windows (for people who don't have tools for extracting these archives), hence the `.exe` suffix. However, you can view the contents as a 7zip archive with tools like 7zip an p7zip.

**Reply by Andystew94** [reply]
Makes sense :) Cheers!
